packages/remirror__extension-doc/src/doc-extension.ts

```diff
@@ -57,7 +59,7 @@ export interface DocOptions {
    *
    * The attributes are directly applied to the dom node.
```

Comment:
Maybe we need a little comment here. To explain what's the difference between array and object.